samples/client/openapi_features/security/python/docs/paths/path_with_security_from_root/get.md

```diff
@@ -128,11 +134,37 @@ security_scheme_info: api_configuration.SecuritySchemeInfo = {
     ),
 }
 
+
+# security_scheme_info for security_index 4
+security_scheme_info: api_configuration.SecuritySchemeInfo = {
+    "oauthClientCredentials": security_scheme_oauth_client_credentials.OauthClientCredentials(
+        flows = security_scheme_oauth_client_credentials.OAuthFlows(
+            client_credentials=security_scheme_oauth_client_credentials.ClientCredentialsOauthFlow(
+            )                       
+        )
+    ),
+}
```

Comment:
How about not including security_scheme_info examples for clientCredentails/implicit/authorizationCode because they have not yet been added to the client?

Reply:
You mean generating this only for the Password one and let the other ones to be implemented in the future??

Reply:
Yes that's right